### ISSUE TITLE
PSR2: Allow multiline defaults in method definition

### ISSUE THREAD

**Issue by aminin** [squizlabs/PHP_CodeSniffer#542]
The example

```php
class ClassName
{
    public function foo($bar = [
        1,
        2,
    ])
    {
        // method body
    }
}
```
should not produce any warnings related to multi-line method definition.

Because there is no split argument list.

See section 3.1 of [PSR-2-meta](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide-meta.md#31---multi-line-arguments-09082013) and 4.4 of [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments)

**Comment by YatkinK** [comment]
+1

**Comment by serhiobb** [comment]
++

**Comment by gsherwood** [comment]
The PSR-2 meta is for function calls, not for function definitions, and directly references 4.6 and not 4.4.

But I suspect that is just another in a long list of PSR-2 mistakes, so I'll get PHPCS to allow this.

**Comment by gsherwood** [comment]
I finally got around to making this change. Sorry it took so long.
